packages/errors/src/errors.ts

```diff
@@ -1185,6 +1185,19 @@ export const AllCypressErrors = {
 
         ${fmt.code(code)}`
   },
+  EXPERIMENTAL_USE_DEFAULT_DOCUMENT_DOMAIN_E2E_ONLY: () => {
+    const code = errPartial`
+    {
+      e2e: {
+        experimentalSkipDomainInjection: ['*.salesforce.com', '*.force.com', '*.google.com', 'google.com']
```

Comment:
should I go with something less specific here?

Reply:
I've opened the #24418 issue, and I think this should be changed for the following domains also: *.my.site.com 
taken from https://help.salesforce.com/s/articleView?id=sf.domain_name_enhanced.htm&type=5

Reply:
@alexanderg1982 this should resolve #24418. This line here is more so to recommend users on how to configure the option if the config isn't inside the `e2e` object. I think it's too specific since we just want to recommend where to move the option. When we release this, you should be able to do:

```js
experimentalSkipDomainInjection: [' power-inspiration-1088-dev-ed.scratch.my.site.com']
```

Reply:
@AtofStryker Maybe it would be good to call out this is a recommendation?

Reply:
I'm wondering if we just want to reference the enhanced domain names in the docs?

Reply:
@alexanderg1982 I added the enhanced domain reference in the [docs](https://github.com/cypress-io/cypress-documentation/pull/4955/files#diff-8df10062ef7f8e59d80d177d29301414b5c83d2fb7c52681870439be2de71b80R97) to help guide the correct domains. Thank you for sharing this!